src/app/commands/cmd_about.cpp

```diff
@@ -39,7 +39,7 @@ void AboutCommand::onExecute(Context* context)
   base::UniquePtr<Window> window(new Window(Window::WithTitleBar, "About " PACKAGE));
   Box* box1 = new Box(VERTICAL);
   Grid* grid = new Grid(2, false);
-  Label* title = new Label(PACKAGE " v" VERSION);
+  Label* title = new Label(std::string(PACKAGE) + " v" + VERSION + " (" + HASH + ")");
```

Comment:
Was it really necessary to add the std::string or the concatenations? It seems it would've worked to simply have:
`(PACKAGE " v" VERSION " (" HASH ")")`
Also, GIT_HEAD_SHA1 is a better macro than HASH, especially since config.h gets included *everywhere*.

Reply:
Brand new to C++ so I'm not sure. I tried something similar to `(PACKAGE " v" VERSION " (" HASH ")")` but I was getting errors saying something about not being able to concatenate to different char*. I'll try to get it to work again on Monday.

I'll also the macro name.

Reply:
@felipemanga This is the error I am getting:

```
.../LibreSprite/src/app/commands/cmd_about.cpp:42:36: error: invalid operands of types ‘const char [12]’ and ‘const char [3]’ to binary ‘operator+’
   42 |   Label* title = new Label(PACKAGE + " v" + VERSION + " (" + GIT_HEAD_SHA1_HASH + ")");
```